cli/src/main/resources/help/module.txt

```diff
@@ -10,17 +10,17 @@ SYNOPSIS
                  [--main-class=fully_qualified_class_name]])
                )
 
-  To remove a module
+  To remove a module locally
 
     module remove --name=module_name [--slot=slot_name]
 
 
 DESCRIPTION
 
-  The command is used to add and remove modules.
+  The command is used to add and remove modules locally.
 
   When a module is added, the corresponding to the module name directory
-  structure will be created in the AS7 module repository. The JAR files
+  structure will be created in the WildFly module repository. The JAR files
```

Comment:
Please, no WildFly. It leads to requirements to change it in EAP. Perhaps "installation's module repository" or just "module repository".